### ISSUE TITLE
Move DB files to another server

### ISSUE THREAD

**Issue by nikgul1995** [ClickHouse/ClickHouse#28062]
Hello.
I have DB installed on Ubuntu server.
I want to move DB files on another server (file server with samba).
Will it be work:
1. I create a folder on another server
2. I move folder /var/lib/clickhouse from current server to file server
3. On current server I mount file-server`s folder to /var/lib/clickhouse

**Comment by den-crane** [comment]
Yes it will work. Check that privileges of  the root folder `/var/lib/clickhouse` are not changed.

**Comment by nikgul1995** [comment]
> > file server with samba).
> 
> Performance of Clickhouse will be not good as it was.

The current server and the file-server are vhosts on one KVM physical machine. Will it work good?
